students/im2594.md

```diff
@@ -12,4 +12,15 @@ Create a library which acts as a MongoDB wrapper in JavaScript to perform common
 
 ## Project Contribution
 
-I would like to contribute to open source projects or programming languages such as Protocol Buffer/gRPC and Golang/C++. I will look into the open issue section of these open source project and will attempt to pick an open issue and contribute into it.
+I would like to contribute to open source projects or programming languages such as Protocol Buffer/gRPC and Golang/C++. I will look into the open issue section of these open source project and will attempt to pick an open issue and contribute into it. 
+
+## Github Profile 
+GitHub: [link](https://github.com/monirul1)
+
+GitHub README: [link](https://github.com/monirul1/monirul1/blob/main/README.md)
+
+## Project Proposal
+- [mongo-wrapper](./projects/javascript/mongo-wrapper.md)
```

Comment:
update link path and re-squash 😈

Reply:
Hi Professor @timkpaine , I'm sorry I didn't understand, what do I need to update? this `./projects/javascript/mongo-wrapper.md` seem to be a right path as I want my read.md file to be under `./projects/javascript`?